sgx/core-types/src/report_body.rs

```diff
@@ -260,6 +267,7 @@ impl FromX64 for ReportBody {
 
 impl Hash for ReportBody {
     fn hash<H: Hasher>(&self, state: &mut H) {
+        "mc_sgx_core_types::report_body::ReportBody".hash(state);
```

Comment:
I'm seeing this here, but not seeing a similar thing in `MiscAttribute` for example. Any reason this struct needs the domain separator but the other one does not?

Reply:
That's an oversight, I'll fix.